components/Header/MessageBar.js

```diff
@@ -20,10 +22,37 @@ const Location = styled.p`
   `};
 `;
 
+const StyledLink = styled.a`
+  margin-left: 2rem;
+  color: ${({ theme }) => theme.colors.white};
+  // text-decoration: underline;
+  border-bottom: solid 1px ${({ theme }) => theme.colors.white};
+
+  &:hover {
+    color: ${({ theme }) => theme.colors.highlight};
+    border-bottom: solid 1px ${({ theme }) => theme.colors.highlight};
+    cursor: pointer;
+  }
+`;
+
 const MessageBar = ({ className }) => {
+  const clickTracking = () => {
+    gtag.event({
+      clientWindow: window,
+      action: 'click',
+      category: 'link button',
+      label: 'message bar link',
+    });
+  };
+
   return (
     <div className={className}>
-      <Message>Call for Speakers starts January 1st!</Message>
+      <Message>
+        Call for Speakers starts January 1st!
```

Comment:
Could we do the following text instead:

> Call for Counselors (Speakers) starts January 1st!

Just trying to eliminate as many instances of "speakers" as possible except in those areas where an uninformed person would want to know WTH a "counselor" is like this banner link area.

Reply:
Good call! updated 🙂